### PR TITLE
Fix Develco SPLZB-131 plug device temperature divisor

### PR DESCRIPTION
**Pull request by eriknn**
Adds quirk for SPLZB-131!

Fixes https://github.com/zigpy/zha-device-handlers/issues/2121

### CODE REVIEW

File: zhaquirks/develco/power_plug.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+
+class DeviceTemperatureCluster(CustomCluster, DeviceTemperature):
+    """Device Temperature. Modify divisor."""
+
+    cluster_id = DeviceTemperature.cluster_id
+
+    def __init__(self, *args, **kwargs):
+        """Init."""
+        super().__init__(*args, **kwargs)
```
Comment (TheJulianJES) [inline zhaquirks/develco/power_plug.py:37]: This can be removed

File: zhaquirks/develco/power_plug.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+from zhaquirks.develco import DEVELCO
+
+
+class DeviceTemperatureCluster(CustomCluster, DeviceTemperature):
+    """Device Temperature. Modify divisor."""
+
+    cluster_id = DeviceTemperature.cluster_id
```
Comment (TheJulianJES) [inline zhaquirks/develco/power_plug.py:33]: This can be removed

**Comment by TheJulianJES** [comment]
Thanks for the PR! I've added two small comments. Please try and see if the quirk still works after making those changes.

**Comment by eriknn** [comment]
Thanks for the feedback! This is my first commit to a community project, so it's a steep "github learning curve" for me here.
Not sure if I understood your comments correctly. Did you just mean the _init function?

**Comment by TheJulianJES** [comment]
All good! Thanks for doing the PR!

I think the explicit cluster_id definition can also be removed, as it should be inherited from the `DeviceTemperatureCluster` already.

File: zhaquirks/develco/power_plug.py [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+        if voltage > 0 and current > 0:
+            voltage = voltage / 100
+            current = current / 1000
+            pf = 100 * power / (voltage * current)
+
+            if pf > 100:
+                pf = 100
+            elif pf < 0:
+                pf = 0
+        self._update_attribute(self.POWERFACTOR_ID, pf)
```
Comment (TheJulianJES) [inline zhaquirks/develco/power_plug.py:73]: Can we leave this manual power factor calculation out for now?
I know ZHA creates a "power factor" sensor by default (even though the device does not support it). This happens for a lot of devices and needs to be fixed in ZHA.
We just don't do this manual calculation for any other devices and I wouldn't do start with it for now.

Comment (eriknn) [reply zhaquirks/develco/power_plug.py]: Sure, new pull request created!

File: zhaquirks/develco/power_plug.py [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+        ENDPOINTS: {
+            1: {
+                PROFILE_ID: 0xC0C9,
+                DEVICE_TYPE: 1,
+                INPUT_CLUSTERS: [
+                    Scenes.cluster_id,
+                    OnOff.cluster_id,
+                ],
+                OUTPUT_CLUSTERS: [],
+            },
```
Comment (TheJulianJES) [inline zhaquirks/develco/power_plug.py:55]: I'm guessing the removed endpoint 1 in the replacement did not have a function at all?

File: zhaquirks/develco/power_plug.py [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+from zhaquirks.develco import DEVELCO
+
+DEV_TEMP_ID = DeviceTemperature.attributes_by_name["current_temperature"].id
+
+
+class DevelcoDeviceTemperature(CustomCluster, DeviceTemperature):
+    """Device Temperature. Modify divisor."""
+
+    def _update_attribute(self, attrid, value):
+        if attrid == DEV_TEMP_ID and value is not None:
```
Comment (TheJulianJES) [inline zhaquirks/develco/power_plug.py:36]: Did you ever get `None` as a value for `_update_attribute`?

Comment (eriknn) [reply zhaquirks/develco/power_plug.py:36]: No, that check might be unnecessary.....

Comment (TheJulianJES) [reply zhaquirks/develco/power_plug.py]: Removed it for now.

**Comment by TheJulianJES** [comment]
Everything should be good now. I've added a small test for the multiplication too.

Thanks for the PR!

**Pull request merged**